app/controllers/fed_ax_api_controller.rb

```diff
@@ -0,0 +1,128 @@
+require "#{ Rails.root }/lib/shipping_api.rb"
+
+class FedAxApiController < ApplicationController
+  include FedAxApiWrapper
+
+  def quote
+    begin
+      content = ShippingApi.query(shipping_quote_params)
+
+      # OPTIMIZE: is this the best way to handle response status from the API Wrapper?
+      if content[:status] == 200
+        status = :ok
+      else content[:status] == 204
+        status = :no_content
+      end
```

Comment:
For optimization purposes it looks like content[:status] could come in as 200 or be set as 200 down on line 24.
You could probably take out line 26 and move this whole section down and it would save a line. 

In a broader sense of the question I'm not entirely certain. The questions that I would have to guide my thinking is asking myself "Can I pass in :ok and :no_content from the wrapper itself?" and "Is there a way to get these codes automatically translated?"